### PR TITLE
Changes to suite file for mobile UCR

### PR DESCRIPTION
**Pull request by nickpell**
http://manage.dimagi.com/default.asp?167204#BugEvent.1002723

### CODE REVIEW

File: corehq/apps/app_manager/models.py [hunk 1]
```diff
@@ -3376,45 +3376,54 @@ def _column_to_series(column):
...
+                        suite_xml.Field(
+                            header=suite_xml.Header(
+                                text=suite_xml.Text(
+                                    locale=suite_xml.Locale(id=id_strings.report_last_sync())
+                                )
+                            ),
+                            template=suite_xml.Template(
+                                text=suite_xml.Text(
+                                    xpath=suite_xml.Xpath(
+                                        function="format-date(date(instance('reports')/reports/@last_sync), '%Y-%m-%d %H:%M')"
```
Comment (dimagimon) [inline corehq/apps/app_manager/models.py]: E501 line too long (126 > 115 characters)

**Comment by nickpell** [comment]
@dannyroberts @orangejenny 

Easiest to read (and see changes reflected in tests) commit by commit

**Comment by nickpell** [comment]
:white_check_mark: tests passed

**Comment by dannyroberts** [comment]
(you still have the don't merge label on. Mistake?)

**Comment by nickpell** [comment]
@orangejenny updated

**Comment by orangejenny** [comment]
looks good! will merge once mergeable

**Comment by nickpell** [comment]
:white_check_mark: mergeable

**Pull request merged**